pkg/controllers/dlb/controller.go

```diff
@@ -35,6 +35,8 @@ import (
 
 const ownerKey = ".metadata.controller.dlb"
 
+var defaultNodeSelector map[string]string = deployments.DLBPluginDaemonSet().Spec.Template.Spec.NodeSelector
```

Comment:
do we really need a type here. Wouldn't := work?

Reply:
I already tried before you commented, but it doesn't..! So, I kept as it is now.!

Reply:
works for me [just fine](https://go.dev/play/p/pRmsp9s0M4m)

Reply:
@bart0sh Umm, I don't understand why the code in the playground is related to what you suggested here..!
Isn't what you suggested more like [this](https://go.dev/play/p/gWAIw-JiSRP)?

Reply:
> do we really need a type here
I suggested to get rid of redundant type declaration. The way of doing it doesn't matter much.

Reply:
@bart0sh
Ok..!
> var defaultNodeSelector map[string]string = deployments.DLBPluginDaemonSet().Spec.Template.Spec.NodeSelector
=>
defaultNodeSelector map[string]string := deployments.DLBPluginDaemonSet().Spec.Template.Spec.NodeSelector

is what you suggested, right?
So, I just showed in the playground that
> foo := "bar"

does not work..!

Reply:
Yes, I suggested to drop redundand type declaration and showed and example how to do it. Please do.

Reply:
I guess Ed is asking for this
```suggestion
var defaultNodeSelector = deployments.DLBPluginDaemonSet().Spec.Template.Spec.NodeSelector
```

Reply:
@bart0sh Umm, sorry, what am I missing?
> Wouldn't := work?

[It does not work..?](https://go.dev/play/p/qtalKnkIR2q)

Did you mean to remove map[string]string?

Reply:
@rojkov Ah, sorry, ":=" confused me so much.
I now understand what he said and see that he said to remove "type declaration."
Thanks for the expalation.!

Reply:
@rojkov thank you for helping me with this :)